resources/policies/external-dns-annotation/ingress_external_dns_identifier_format.rego

```diff
@@ -0,0 +1,11 @@
+package cloud_platform.admission
+
+# This policy deny any ingress that don't have annotation "external-dns.alpha.kubernetes.io/set-identifier"
+# or not use 'cloud-platform.justice.gov.uk/ignore-external-dns-weight: "true"' annotation to ignore about check.
+
+deny[msg] {
+  input.request.kind.kind == "Ingress"
+  not input.request.object.metadata.annotations["external-dns.alpha.kubernetes.io/set-identifier"] == concat("-", [input.request.object.metadata.name, input.request.object.metadata.namespace, ${cluster_color}])
```

Comment:
@mogaal , as discussed, can we check for label `cloud-platform.justice.gov.uk/environment-name` instead of namespace name. The format we decided is <ingress-name>-<environment-name from label>-<color>

Reply:
Finalized with the team to have <ingress-name>-<ns-name>-<color>